### PR TITLE
Adding More Emojis

### PR DESCRIPTION
**Pull request by bhavesh1129**
Hello!
I had added more Emoji's it would be great for me if you accept my PR as this is my first **Hacktoberfest**.
Thankyou!

### CODE REVIEW

**Review by RishabhDevbanshi — APPROVED** [review]
All Good To Go ! Will merge it by today .

**Comment by bhavesh1129** [comment]
> All Good To Go ! Will merge it by today .

Thankyou soo much!

File: src/emojipedia.js [hunk 1]
```diff
@@ -1,93 +1,154 @@
...
+        id: 10,
+        emoji: "🙈",
+        name: " See-No-Evil Monkey",
+        meaning: "The see no evil monkey, called Mizaru (Japanese for “see not”), one of the Three Wise Monkeys. Depicted as the brown 🐵 Monkey Face with tan or pinkish hands covering its eyes.“."
+    },
+    {
+        id: 11,
+        emoji: "🤮",
+        name: "Face Vomiting",
+        meaning: "A yellow face with scrunched eyes spewing birght-green vomit. This emoji represents illness or disgust.“."
```
Comment (TheLearneer) [inline src/emojipedia.js:65]: Why add a extra “ just before the end. Also same issue with few other emojis

File: src/emojipedia.js [hunk 2]
```diff
@@ -1,93 +1,154 @@
...
+        id: 2,
+        emoji: "🙏",
+        name: "Person With Folded Hands",
+        meaning: "Two hands pressed together. Is currently very introverted, saying a prayer, or hoping for enlightenment. Is also used as a “high five” or to say thank you."
+    },
+    {
+        id: 3,
+        emoji: "🤣",
+        name: "Rolling On The Floor, Laughing",
+        meaning: "This is funny! A smiley face, rolling on the floor, laughing. The face is laughing boundlessly. The emoji version of “rofl“. Stands for „rolling on the floor, laughing“."
```
Comment (TheLearneer) [inline src/emojipedia.js:17]: incorrect use of quotation mark

File: src/emojipedia.js [hunk 3]
```diff
@@ -1,93 +1,154 @@
...
+    },
+    {
+        id: 4,
+        emoji: "😎",
+        name: "Face With Sunglasses",
+        meaning: "I'm the king of the world! Mr. Cool Smiley with sunglasses. Is totally self-assured and relaxed. Everything is absolutely easy and cool. Also a symbol for sun, summer holidays, and holidays on the beach.“."
```
Comment (TheLearneer) [inline src/emojipedia.js:23]: incorrect use of quotation mark

File: src/emojipedia.js [hunk 4]
```diff
@@ -1,93 +1,154 @@
...
+    },
+    {
+        id: 5,
+        emoji: "😉",
+        name: "Winking Face",
+        meaning: "A yellow face with a slight smile shown winking, usually its left eye. May signal a joke, flirtation, hidden meaning, or general positivity. Tone varies, including playful, affectionate, suggestive, or ironic.“."
```
Comment (TheLearneer) [inline src/emojipedia.js:29]: incorrect use of quotation symbol

**Review by bsantosh909 — CHANGES_REQUESTED** [review]

**Comment by bhavesh1129** [comment]
> All Good To Go ! Will merge it by today .

Hey!!
You haven't merged it yet.
P.S: A small Reminder :)

**Review by bhavesh1129 — COMMENTED** [review]
Thanks for correcting

**Pull request closed without merging**